Website/package.json

```diff
@@ -13,6 +13,7 @@
     "autoprefixer": "10.4.16",
     "eslint": "8.50.0",
     "eslint-config-next": "13.5.3",
+    "font-awesome": "^4.7.0",
```

Comment:
@anevski-stefan Please remove the `font-awesome` module as it is no longer required.
```suggestion
```